package.json

```diff
@@ -43,7 +43,7 @@
     "test": "npm run lint && npm run test-unit",
     "build": "rollup -c utils/build/rollup.config.js",
     "build-module": "rollup -c utils/build/rollup.config.js --configOnlyModule",
-    "dev": "concurrently --names \"ROLLUP,HTTP\" -c \"bgBlue.bold,bgGreen.bold\" \"rollup -c utils/build/rollup.config.js -w -m inline\" \"servez -p 8080 --ssl\"",
+    "dev": "concurrently -P --names \"ROLLUP,HTTP\" -c \"bgBlue.bold,bgGreen.bold\" \"rollup -c utils/build/rollup.config.js -w -m inline\" \"servez -p 8080 --ssl {@}\"",
```

Comment:
this is to allow overriding `servez` options, for `--ssl=false` in https://github.com/mrdoob/three.js/pull/26055/files#diff-24ad71c8613ddcf6fd23818cb3bb477a1fb6d83af4550b0bad43099813088686R9

Reply:
Why do we need `--ssl=false`?

Reply:
`--ssl=false` is only for inside the codespace, on localhost, ssl will be on like before

Reply:
the reason is that the endpoint Codespace mounts for the preview is already configured configured with ssl